### PR TITLE
Reword EVM section on ethereum stack page (Chinese)

### PR DESCRIPTION
**Pull request by gongxh13**

### CODE REVIEW

**Comment by minimalsm** [comment]
Thanks @gongxh13,

@Riely007 can you check this please 🙏

File: src/content/translations/zh/developers/docs/ethereum-stack/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ sidebar: true
 
 ## 级别 1：以太坊虚拟机 {#ethereum-virtual-machine}
 
-[以太空虚拟机 (EVM)](/developers/docs/evm/) 是用于智能合约的运行环境。 Ethereum 区块链上的所有智能合约和状态更改都由[交易](/developers/docs/transactions/)执行。 EVM 处理以在太坊网络上执行的所有交易。
+[以太坊虚拟机 (EVM)](/developers/docs/evm/) 是用于智能合约的运行环境。 Ethereum 区块链上的所有智能合约和状态更改都由[交易](/developers/docs/transactions/)执行。 EVM 处理以在太坊网络上执行的所有交易。
```
Comment (Riely007) [inline src/content/translations/zh/developers/docs/ethereum-stack/index.md:14]: [以太坊虚拟机 (EVM)](/developers/docs/evm/) 是以太坊智能合约的运行环境。 以太坊区块链上所有智能合约的状态都通过执行[交易](/developers/docs/transactions/)来改变的。 

Will this be better? 😄

Also, can you confirm this one -"The EVM handles all of the transaction processing on the Ethereum network."? Seems

Comment (Riely007) [reply src/content/translations/zh/developers/docs/ethereum-stack/index.md:14]: Also, can you confirm this one -"The EVM handles all of the transaction processing on the Ethereum network."? Seems not all the transactions on the Ethereum network need EVM, eg: EOA -> EOA, ether transactions won't be executed by EVM.  🫣 @minimalsm

Comment (minimalsm) [reply src/content/translations/zh/developers/docs/ethereum-stack/index.md:14]: @Riely007 yeah, I think you're right... 

Not a huge deal that we need to deal with here but I'll create a card to update the English source and that can filter into other languages.

Comment (minimalsm) [inline src/content/translations/zh/developers/docs/ethereum-stack/index.md:14]:
```suggestion
[以太坊虚拟机 (EVM)](/developers/docs/evm/) 是以太坊智能合约的运行环境。 以太坊区块链上所有智能合约的状态都通过执行[交易](/developers/docs/transactions/)来改变。 
```

**Review by minimalsm — APPROVED** [review]

**Pull request merged**